<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtPerf</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInfo">
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonReset">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelRounds">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Test rounds:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinRounds">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="comboTests">
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QLineEdit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QComboBox</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QComboBoxEntry</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QSpinBox</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QProgressBar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QPushButton</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QCheckBox</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QRadioButton</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QTextEdit - Add text</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QTextEdit - Scroll</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QPainter- Lines</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QPainter - Circles</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QPainter - Text</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QPainter - Image</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="buttonRun">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textConsole"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEntrys">
       <attribute name="title">
        <string>Entrys</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="testLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="testComboBox">
          <item>
           <property name="text">
            <string>Selection 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 10</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="testComboBoxEd">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Selection 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection 10</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="testSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QProgressBar" name="testProgressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabButtons">
       <attribute name="title">
        <string>Buttons</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>82</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="testButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>184</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="testCheckBox">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" rowspan="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>184</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="QRadioButton" name="testRadioButton1">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QRadioButton" name="testRadioButton2">
          <property name="text">
           <string>RadioButton</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>81</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabText">
       <attribute name="title">
        <string>Text</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="testTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDrawing">
       <attribute name="title">
        <string>Drawing</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelQPainter">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>QPainter</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="TestWidget" name="testDrawWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TestWidget</class>
   <extends>QWidget</extends>
   <header>testwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
